vllm/model_executor/models/qwen2_moe.py

```diff
@@ -312,6 +313,7 @@ def forward(
         return hidden_states, residual
 
 
+@support_torch_compile
```

Comment:
this will actually cause on error, but let me merge it first, so that pytorch team can easily reproduce it.

Reply:
the error is:

`RuntimeError: aten/src/ATen/RegisterCUDA.cpp:7275: SymIntArrayRef expected to contain only concrete integers`